samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Tests.razor

```diff
@@ -8,6 +8,7 @@
 <button disabled="@(!_mapRendered)" @onclick="() => ToggleLayer(_graphicsLayer)">Graphics Layer</button>
 <button disabled="@(!_mapRendered)" @onclick="() => ToggleLayer(_geoJsonLayer)">GeoJSON Layer</button>
 <button disabled="@(!_mapRendered)" @onclick="() => ToggleLayer(_geoRssLayer)">GeoRSS Layer</button>
+<button disabled="@(!_mapRendered)" @onclick="() => ToggleLayer(_csvLayer)">CSV Layer</button>
```

Comment:
Can you create a full samples page for this new layer type? It helps with the discoverability, and this tests page is a mess that I don't publish to our `samples.geoblazor.com` site.